sdk/nodejs/package.json

```diff
@@ -31,7 +31,6 @@
         "typescript": "^4.3.5"
     },
     "pulumi": {
-        "resource": true,
-        "name": "kubernetes"
```

Comment:
This doesn't look right to me. Any idea why this changed?

Reply:
I think I didn't run the correct `make` targets. Re-ran, force-pushed, and the diff is now gone.